pkg/monitor/monitor/util.go

```diff
@@ -742,6 +723,18 @@ func getMonitorVolumes(config *core.ConfigMap, monitor *v1alpha1.TidbMonitor, tc
 		volumes = append(volumes, monitor.Spec.AdditionalVolumes...)
 	}
 
+	// add asset tls
```

Comment:
```suggestion
	// Add TLS assets secret volume by default, so that no need to rolling update the monitor
	// Pod if users add new TLS clusters even if non of the original clusters has TLS enabled
```